test/functionality/abstract-interpretation/data-frame/inference.test.ts

```diff
@@ -852,11 +1024,11 @@ df <- transform(df, name = c(letters[1:5]))
 		]
 	);
 
-	describe.skipIf(skipDplyr)('dplyr Functions', () => {
+	describe.skipIf(skipLibraries)('dplyr Functions', () => {
 		testDataFrameDomain(
 			`
 df <- data.frame(id = 1:5, score = c(80, 75, 90, 70, 85))
-df <- dplyr::group_by(df, id) |> as.data.frame()
+df <- dplyr::group_by(df, id)
```

Comment:
do we drop this because we do not handle the cast? or because we dropped the separation?
I mean one safe way for `as.data.frame` to work would be to use identity if we already have df shape info